pyuploadcare/api/api.py

```diff
@@ -378,10 +381,22 @@ def upload_from_url(
                 self.secret_key, expire  # type: ignore
             )
 
+        if check_duplicates is not None:
+            data["check_URL_duplicates"] = "1" if check_duplicates else "0"
+
+        if save_duplicates is not None:
+            data["save_URL_duplicates"] = "1" if save_duplicates else "0"
+
         url = self._build_url(base="/from_url")
         document = self._client.post(url, data=data)
         response = document.json()
         if "token" not in response:
+            if check_duplicates and response["type"] == "file_info":
```

Comment:
- [ ] Is there a better way to check that?

Reply:
i have no idea